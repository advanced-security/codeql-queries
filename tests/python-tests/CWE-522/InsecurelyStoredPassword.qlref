CWE-522/InsecurelyStoredPassword.ql