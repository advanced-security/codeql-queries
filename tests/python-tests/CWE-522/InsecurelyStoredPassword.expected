| insecure_pw_test.py:14:1:14:40 | ClassDef | Insecure 'user' class $@ stores its password insecurely, without secure hashing | insecure_pw_test.py:14:1:14:40 | ClassDef | InsecureUser |
| insecure_pw_test.py:21:1:21:41 | ClassDef | Insecure 'user' class $@ stores its password insecurely, without secure hashing | insecure_pw_test.py:21:1:21:41 | ClassDef | InsecureUser2 |
| insecure_pw_test.py:33:1:33:41 | ClassDef | Insecure 'user' class $@ stores its password insecurely, without secure hashing | insecure_pw_test.py:33:1:33:41 | ClassDef | InsecureUser3 |
